modules/op-installing-pipelines-as-code-on-an-openshift-cluster.adoc

```diff
@@ -0,0 +1,74 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="installing-pipelines-as-code-on-an-openshift-cluster_{context}"]
+= Installing {pac} on an {product-title}
+
+[role="_abstract"]
+Starting with {pipelines-title} 1.7, {pac} is installed by default when you install the {pipelines-title} Operator. If you are using {pipelines-shortname} 1.7 or later versions, skip the procedure for manual installation of {pac}.
+
+However, if you want to disable the default installation of {pac} with the {pipelines-title} Operator, set the value of the `enablePipelinesAsCode` field as `false`k in the `TektonConfig` custom resource.
+
+[source,yaml]
+----
+...
+spec:
+    addon:
+      enablePipelinesAsCode: false
+...
+----
+
+[discrete]
+.Procedure
+
+. To manually install {pac} on a {product-title} cluster instead of the default installation with the {pipelines-title} Operator, run the following command:
```

Comment:
do we need to keep manual install now? as it is possible through operator?
cc @chmouel

Reply:
if user change `enablePipelinesAsCode` to false and later decide to install it again, then they just need to change it back to true